static/app/constants/ios-device-list.tsx

```diff
@@ -1,155 +0,0 @@
-// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-// generated using scripts/extract-ios-device-names.ts as part of build step.
-// the purpose of the script is to extract only the iOS information that Sentry cares about
-// and discard the rest of the JSON so we do not end up bloating bundle size.
-
-const iOSDeviceMapping: Record<string, string> = {
```

Comment:
Not sure why this is in the diff but it's gone from git now since it's an autogenerated file

Reply:
Just `git checkout static/app/constants/ios-device-list.tsx` here maybe?